### DISCUSSION TITLE
Changing the company field from input field to textarea still possible ?

### DISCUSSION THREAD

**Discussion by micha68g** [oliverklee/ext-onetimeaccount#629] [Q&A] [answered]
Hi,

is there still a way to change the company field from an <input-field to <textarea> ? (like in ver. 3.x for example)
Back then it was something like:
plugin.tx_onetimeaccount_pi1.form.elements {
  company = renderlet:TEXTAREA
  company.custom = rows="3" cols="20"
}

Didn't find anything in the docs.

Thanks
Michael

**Comment by oliverklee** [comment, marked as answer]
Yes, you can use a custom Fluid template and then use a different view helper.

In your TypoScript constants, use this:

```typoscript
plugin.tx_onetimeaccount {
  view {
    # cat=plugin.tx_onetimeaccount/file; type=string; label=Path to template partials (FE)
    partialRootPath = EXT:my_extension/Resources/Private/Partials/
  }
}
```

Then copy the `User.html` partial and modify it.

**Reply by micha68g** [reply]
thanks.... !
